src/layouts/Base.astro

```diff
@@ -80,39 +82,27 @@ const pathname = Astro.url.pathname.replace(/\/$/, "")
 				</div>
 			</BackgroundGridContainer>
 
-			<main
-				class="container flex flex-col pt-8 sm:px-6 sm:pt-12 lg:px-8 lg:pt-16"
-			>
-				<ul class="templates mb-24 flex flex-col gap-24">
+			<main class="container">
+				<TabPanel>
 					{
-						groups.map(([category, examples]) => {
-							return (
-								<li class="flex flex-col gap-8">
-									<h2
-										class:list={[
-											"text-2xl",
-											{ "sr-only": category === TOP_SECTION },
-										]}
-									>
-										{category}
-									</h2>
-									<ul class="grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6">
-										{examples.map(
-											(
-												example: astroHTML.JSX.IntrinsicAttributes & CardProps,
-											) => (
-												<li>
-													<Card {...example} />
-												</li>
-											),
-										)}
-										{category === TOP_SECTION && <Outlinks />}
-									</ul>
-								</li>
-							)
-						})
+						groups.map(([category], index) => (
+							<Tab id={category} label={category} default={index === 0} />
+						))
 					}
-				</ul>
+				</TabPanel>
+				{
+					groups.map(([category, examples], index) => (
+						<TabContent id={category} default={index === 0}>
+							<ul class="mt-8 grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6">
+								{examples.map((example) => (
+									<li>
+										<Card {...example} />
+									</li>
+								))}
+							</ul>
+						</TabContent>
+					))
+				}
```

Comment:
@tony-sull I'm not a fan of the duplicate `default` on `Tab` and `TabContent`, wdyt? I initially tried something like this, but it caused a syntax error (or I did something else wrong):
```astro
<Tabs
  tabs={[
    { id: 'tab1', label: 'Tab 1', content: <p>content 1</p> },
    { id: 'tab2', label: 'Tab 2', content: <p>content 2</p> },
  ]}
/>
```

Reply:
We synced offline but just to loop back here, `getStaticPaths` in a `pages/[tab].astro` route should do the trick here since we aren't doing any kind of client routing